### ISSUE TITLE
[BUG] Privado doesn't have write permission on give repo location - /app/code

### ISSUE THREAD

**Issue by juicetin** [Privado-Inc/privado#416]
**Describe the bug**
Privado doesn't run out of the box (perhaps missing docs)

**To Reproduce**
1. install the cli as per readme - `curl -o- https://raw.githubusercontent.com/Privado-Inc/privado-cli/main/install.sh | bash`
2. clone the sample repo as per readme - `git clone git@github.com:saurabh-sudo/BankingSystem-Backend.git`
3. attempt to scan the repo - `privado scan BankingSystem-Backend`

**Expected behavior**
* scan should execute out of the box

**Screenshots**
![image](https://github.com/Privado-Inc/privado/assets/8791327/c01e08a5-8f7a-4dc7-9be3-8038a660b451)

**Desktop (please complete the following information):**
 - OS: Fedora Linux 39 Workstation Edition
 - Version Privado CLI: Version v2.2.9 (linux-amd64)

**Additional context**

**Comment by ojaswa1942** [comment]
Hi @juicetin 
The error indicates that the tool doesn't have write permissions on the directory being scanned, which raises an error. Privado CLI stores the result in `.privado` directory inside the scanned directory (repository) and needs write permissions to do so.

**Comment by juicetin** [comment]
Thank you @ojaswa1942 !

I realised that after playing around with the directory, and creating a `.privado` directory in the repo being scanned changed the error to not being able to write to `.privado/debug.log`.

I'm currently still trying to figure this one out - I double checked users and groups were working (docker compose apps in other repos were able to write to local bound volumes) and couldn't fix it down that route. Just for testing/debugging purposes, I tried the nuclear option of recursively setting chmod 777 on everything on and in the directory but still had the same write issues in this case.
